apps/mull-ui/src/utilities.ts

```diff
@@ -96,6 +96,20 @@ export const hasEmoji = (text: string) => {
   return !!emojiRegexRGI().exec(text);
 };
 
+export const exceedsNbOfLines = (descriptionLines: string) => {
+  if (typeof descriptionLines !== 'undefined' && descriptionLines.length > 1) {
+    let countLines = 1;
+    for (let i = 0; i < descriptionLines.length; i++) {
+      if (descriptionLines.charAt(i) === '\n') countLines++;
+    }
+    if (countLines > 6) {
+      return false;
+    }
+    return true;
+  }
+  return true;
```

Comment:
💡This doesn't need to be this complicated:
1. Instead of checking if a variable has the type `"undefined"`, its better to check if the variable is undefined
2. You can replace all of this with a regex call:

```suggestion
  // True if string falsy or <= 6 lines
  return !string || string.match(/\n/gi).length <= 6;
```

Reply:
![image](https://user-images.githubusercontent.com/23544999/113986086-d977a280-981a-11eb-844d-cf565dca10dd.png)

Reply:
and like I mentioned something this simple might not need its own function